translations/ga/census_individual_gb_nir_ga.po

```diff
@@ -2,3312 +2,4784 @@ msgid ""
-msgstr "Tá {person_name} {age_in_years} bliain d'aois. An bhfuil sé seo ceart?"
+#. Question text
+msgid "What is the address of your main place of <em>work</em>?"
+msgstr "Cad é an seoladh atá ag do phríomháit <em>oibre</em>?"
 
-msgid "What is <em>{person_name_possessive}</em> sex?"
-msgstr "Cad í inscne <em>{person_name_possessive}</em>?"
+#. Question text
+msgid "What is the address of <em>{person_name_possessive}</em> main place of <em>work</em>?"
+msgstr "Cad é an seoladh atá ag príomháit <em>oibre</em> <em>{person_name_possessive}</em>?"
+
+#. Question text
+msgid "What was the address of your main place of <em>work</em>?"
+msgstr "Cad é an seoladh a bhí ag do phríomháit <em>oibre</em>?"
+
+#. Question text
+msgid "What was the address of <em>{person_name_possessive}</em> main place of <em>work</em>?"
+msgstr "Cad é an seoladh a bhí ag príomháit <em>oibre</em> <em>{person_name_possessive}</em>?"
+
+#. Question text
+msgid "How do you usually travel to your main place of <em>work</em>?"
+msgstr "Cad é mar a thaistealaíonn tú go dtí do phríomháit <em>oibre</em> de ghnáth?"
+
+#. Question text
+msgid "How does <em>{person_name}</em> usually travel to their main place of <em>work</em>?"
+msgstr "Cad é mar a thaistealaíonn <em>{person_name}</em> go dtí n-áit <em>oibre</em> de ghnáth?"
+
+#. Question text
+msgid "How did you usually travel to your main place of <em>work</em>?"
+msgstr "Cad é mar a thaistil tú go dtí do phríomháit <em>oibre</em> de ghnáth?"
+
+#. Question text
+msgid "How did <em>{person_name}</em> usually travel to their main place of <em>work</em>?"
+msgstr "Cad é mar a thaistil <em>{person_name}</em> go dtí n-áit <em>oibre</em> de ghnáth?"
+
+#. Question text
+msgid "Where is your main place of <em>study</em>?"
+msgstr "Cá bhfuil do phríomháit<em>staidéir</em>?"
 
-msgid "What is <em>{person_name_possessive}</em> marital or same-sex civil partnership status?"
-msgstr "Cad é an stádas pósta nó an stádas páirtnéireachta sibhialta comhghnéis atá ag <em>{person_name_possessive}</em>?"
+#. Question text
+msgid "Where is <em>{person_name_possessive}</em> main place of <em>study</em>?"
+msgstr "Cá bhfuil príomháit<em>staidéir</em><em>{person_name_possessive}</em>?"
+
+#. Question text
+msgid "Is your place of <em>study</em> in Northern Ireland?"
+msgstr "An bhfuil do phríomháit <em>staidéir</em> i dTuaisceart Éireann?"
+
+#. Question text
+msgid "Is <em>{person_name_possessive}</em> place of <em>study</em> in Northern Ireland?"
+msgstr "An bhfuil <em>{person_name_possessive}</em> áit<em>staidéir</em> i dTuaisceart Éireann?"
+
+#. Question text
+msgid "In which country is your course of <em>study</em>, including school?"
+msgstr "Cén tír ina bhfuil do chúrsa <em>staidéir</em>, an scoil san áireamh?"
+
+#. Question text
+msgid "In which country is <em>{person_name_possessive}</em> course of <em>study</em>, including school?"
+msgstr "Cén tír ina bhfuil cúrsa<em>study</em><em>{person_name_possessive}</em>, scoil san áireamh?"
```

Comment:
```suggestion
msgstr "Cén tír ina bhfuil cúrsa <em>study</em> <em>{person_name_possessive}</em>, scoil san áireamh?"
```
Possible missing translation of "study".

Reply:
changed